From a0991727545967d9e4b54a16f12ae641991a1e76 Mon Sep 17 00:00:00 2001
From: Sam Lantinga <slouken@libsdl.org>
Date: Tue, 3 Jan 2023 15:23:38 -0800
Subject: [PATCH 134/458] Added support for the ThrustMaster eSwap PRO
 Controller Xbox

(cherry picked from commit 0357390fc2dcc7c772ac3041c1148882f6fe2137)
---
 .../app/src/main/java/org/libsdl/app/HIDDeviceManager.java       | 1 +
 1 file changed, 1 insertion(+)

diff --git a/android-project/app/src/main/java/org/libsdl/app/HIDDeviceManager.java b/android-project/app/src/main/java/org/libsdl/app/HIDDeviceManager.java
index e695a1772..b5e243f33 100644
--- a/android-project/app/src/main/java/org/libsdl/app/HIDDeviceManager.java
+++ b/android-project/app/src/main/java/org/libsdl/app/HIDDeviceManager.java
@@ -272,6 +272,7 @@ public class HIDDeviceManager {
         final int XB1_IFACE_SUBCLASS = 71;
         final int XB1_IFACE_PROTOCOL = 208;
         final int[] SUPPORTED_VENDORS = {
+            0x044f, // Thrustmaster
             0x045e, // Microsoft
             0x0738, // Mad Catz
             0x0e6f, // PDP
-- 
2.17.1

